conf/master

```diff
@@ -643,6 +643,10 @@
 # as a percent of total actions will be shown for each state run.
 #state_output_pct: False
 
+# The state_compress_ids setting aggregates information about states which have
+# multiple "names" under the same state ID in the highstate output.
+#state_compress_ids: False
```

Comment:
Why not enable this by default?

Reply:
I'd argue that the vast majority of users won't enable this so it'll almost never be used if you don't enable it by default. It's reasonable to ship "sensible" and "nice" defaults, which this falls squarely within. I wouldn't worry about anyone parsing the output of this outputter because that's a stupid thing to do. I don't really see any reason to not enable this by default

Reply:
@frebib since the current output is to not roll this up. the correct thing to do would be to list a deprecation of the current default of it is not being enabled then changing the default when the deprecation is up. that way a fair warning that functionality on the way it outputs currently when names is used will be changing. a lot of people build scripts around the output so any kind of change like that needs proper depreciation.

Reply:
Talked to @waynew about this and I'm going to bring up changing the default at the next Open Hour. If there aren't any compelling reasons to keep this disabled by default, I'll put in another PR to start down the path toward switching the default within a deprecation period.